docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -46,32 +50,46 @@ Google Cloud Pub/Sub API, if different from the ones in the
 https://developers.google.com/identity/protocols/googlescopes[OAuth2 scope] for Spring Cloud GCP
 Pub/Sub credentials | No | https://www.googleapis.com/auth/pubsub
-| `spring.cloud.gcp.pubsub.keepAliveIntervalMinutes` | Determines frequency of keepalive gRPC ping | No | `5 minutes`
+|===
+
+==== Publisher/Subscriber Configuration
+
+This section describes configuration options to customize the behavior of the application's Pub/Sub publishers and subscribers.
+
+|===
+| Name | Description | Required | Default value
 | `spring.cloud.gcp.pubsub.subscriber.parallel-pull-count` | The number of pull workers | No | 1
 | `spring.cloud.gcp.pubsub.subscriber.max-ack-extension-period` | The maximum period a message ack deadline will be extended, in seconds | No | 0
 | `spring.cloud.gcp.pubsub.subscriber.pull-endpoint` | The endpoint for synchronous pulling messages | No | pubsub.googleapis.com:443
 | `spring.cloud.gcp.pubsub.[subscriber,publisher].executor-threads` | Number of threads used by `Subscriber` instances created by `SubscriberFactory` | No | 4
 | `spring.cloud.gcp.pubsub.[subscriber,publisher.batching].flow-control.max-outstanding-element-count`|
-Maximum number of outstanding elements to keep in memory before enforcing flow control.|No | unlimited
+Maximum number of outstanding elements to keep in memory before enforcing flow control.| No | unlimited
 | `spring.cloud.gcp.pubsub.[subscriber,publisher.batching].flow-control.max-outstanding-request-bytes`|
-Maximum number of outstanding bytes to keep in memory before enforcing flow control.|No | unlimited
+Maximum number of outstanding bytes to keep in memory before enforcing flow control.| No | unlimited
 | `spring.cloud.gcp.pubsub.[subscriber,publisher.batching].flow-control.limit-exceeded-behavior`|
-The behavior when the specified limits are exceeded.|No | Block
+The behavior when the specified limits are exceeded.| No | Block
 | `spring.cloud.gcp.pubsub.publisher.batching.element-count-threshold`|
-The element count threshold to use for batching.|No | unset (threshold does not apply)
+The element count threshold to use for batching.| No | unset (threshold does not apply)
 | `spring.cloud.gcp.pubsub.publisher.batching.request-byte-threshold`|
-The request byte threshold to use for batching.|No | unset (threshold does not apply)
+The request byte threshold to use for batching.| No | unset (threshold does not apply)
 | `spring.cloud.gcp.pubsub.publisher.batching.delay-threshold-seconds`|
 The delay threshold to use for batching.
-After this amount of time has elapsed (counting from the first element added), the elements will be wrapped up in a batch and sent.|No | unset (threshold does not apply)
+After this amount of time has elapsed (counting from the first element added), the elements will be wrapped up in a batch and sent. | No | unset (threshold does not apply)
 | `spring.cloud.gcp.pubsub.publisher.batching.enabled`|
-Enables batching.|No | false
+Enables batching. | No | false
```

Comment:
Why the extra space here before the "|" but not in the previous items?

Reply:
Fixed.